packages/state-transition/src/epoch/processProposerLookahead.ts

```diff
@@ -10,12 +11,26 @@ import {computeProposerIndices} from "../util/seed.js";
  * this means that at the start of epoch `N`, the proposer lookahead for epoch
  * `N+1` will be computed and included in the beacon state's lookahead.
  */
-export function processProposerLookahead(fork: ForkSeq, state: CachedBeaconStateFulu): void {
+export function processProposerLookahead(
+  fork: ForkSeq,
+  state: CachedBeaconStateFulu,
+  cache: EpochTransitionCache
+): void {
   // Shift out proposers in the first epoch
   const remainingProposerLookahead = state.proposerLookahead.getAll().slice(SLOTS_PER_EPOCH);
 
   // Fill in the last epoch with new proposer indices
-  const lastEpochProposerLookahead = computeProposerIndices(fork, state, state.epochCtx.epoch + MIN_SEED_LOOKAHEAD + 1);
+  const epoch = state.epochCtx.epoch + MIN_SEED_LOOKAHEAD + 1;
+
+  const shuffling =
+    state.epochCtx.shufflingCache?.getSync(epoch, cache.nextShufflingDecisionRoot, {
```

Comment:
From https://github.com/ChainSafe/lodestar/pull/7945#discussion_r2139659182
> don't need the last param because we handled computeEpochShuffling() below

so the idea is to use exisiting metrics, do we make wanna a label to the metric to differentiate the caller of `getSync` to see which call caused the shuffling miss? @twoeths

Reply:
nvm, we have the `computeEpochShuffling` just because of optional `shufflingCache`. I missed this point.
after fulu, I think we don't need the `getSync` api in `afterProcessEpoch` to avoid the confustion